src/device/llvm.jl

```diff
@@ -0,0 +1,9 @@
+# wrappers for LLVM-specific functionality
+
+@inline trap() = ccall("llvm.trap", llvmcall, Cvoid, ())
```

Comment:
I have this intrinsic plus one other in `src/device/gcn_intrinsics/extra.jl`; would you want to remove that file and move the extra intrinsic (`llvm.debugtrap`) here?